site/_less/layouts/action-network-petitions/white-two-column.less

```diff
@@ -141,11 +145,106 @@
 
   .petition-info {
     @media screen and (min-width: @large-phone) {
-      padding: 0 1rem 0 0;
+      padding: 0 2rem 0 0;
     }
   }
 
+   [for="opt-in"] {
+     background-color: white;
+     color: @blue-800;
+     padding: 2rem 5px;
+  }
+
   .an-footer-cta {
     display: none;
   }
 }
+
+  .petition blockquote p{
+     font-size: 0.8em;
+     color: #ecedfb;
+    line-height: 2rem;
+  }
+ 
+   main.white-two-column h2{
+     margin-bottom: 0.5em;
+   }
+ 
+   .white-two-column [for="opt-in"]{
+     color: #8889a0;
+   }
+ 
+   .petition-info h2{
+     font-weight: 400;
+   }
+ 
+   .petition [type=email], .petition [type=text]{
+     padding: 1.5rem;
+     font-size: 18px;
+   }
+ 
+ input[placeholder], [placeholder], *[placeholder] {
+   color: #4f4f69 !important;
+ }
+ 
+ input::-webkit-input-placeholder {
+    color:  #4f4f69 !important;
+ }
+ 
+ input:-moz-placeholder { /* Firefox 18- */
+    color:  #4f4f69 !important;  
+ }
+ 
+ input::-moz-placeholder {  /* Firefox 19+ */
+    color:  #4f4f69 !important;  
+ }
+ 
+ input:-ms-input-placeholder {  
+    color:  #4f4f69 !important;  
+ }
+ 
+ main.white-two-column{
+   margin-bottom: 9rem;
+ }
+ 
+ main.white-two-column a{
+   color: #2054b3;
+   font-weight: 600;
+ }
+ 
+ .petition p{
+   line-height: 2.9rem;
+   font-size: 2rem;
+ }
+ .stickybar-take-action{
+   font-weight: 900;
+   font-size: 2rem;
+ }
+   .an-footer-info p{
+     line-height: 2.2rem;
+     font-size: 1.7rem;
+   }
+ 
+   .share-this-button-links a{
+     font-weight: 600;
+     letter-spacing: 0.09rem;
+     font-size: 2rem;
+   }
+ 
+   .white-two-column img{
+     -webkit-box-shadow: 0px 14px 40px -22px rgba(139,139,139,1);
```

Comment:
A couple things to learn from this.

First, this should be nested beneath `.white-two-column` above, around L50.
Vendor prefixes get added in the build process, so we don't add them to the source code.
Any number larger than ~ 5px should be declared in rems.
An rgba color with an alpha of 1 can be simplified by using hexadecimal (`#8B8B8B`) or, even better, using less variables like `darken(@grey, 12%)`

Ideally this reads

```less
img {
  box-shadow: 0 1.4rem 4rem -2.2rem darken(@grey, 12%);
}
```